src/actions/actionTypes.js

```diff
@@ -97,22 +97,22 @@ export const POST_RECIPE_OK = "POST_RECIPE_OK";
 export const POST_RECIPE_FAIL = "POST_RECIPE_FAIL";
 
 //EDIT RECIPE
-
-export const EDIT_RECIPE = "EDIT_RECIPE"
-export const EDIT_RECIPE_CATEGORIES="EDIT_RECIPE_CATEGORIES"
-export const EDIT_IMAGES="EDIT_IMAGES"
-export const EDIT_INSTRUCTIONS="EDIT_INSTRUCTIONS"
-// export const EDIT_RECIPE_TAGS="EDIT_RECIPE_TAGS"
-// export const EDIT_RECIPE_INGREDIENTS="EDIT_RECIPE_INGREDIENTS";
-//since recipe_tags & recipe_ingredients are stored for future use
-//we'll rather delete them and add what we'll prefer, rather than
-//edit them from the database  
-
-
-
-//SUBMIT EDITED RECIPE 
 export const EDIT_RECIPE_FAIL="EDIT_RECIPE_FAIL";
 export const EDIT_RECIPE_OK= "EDIT_RECIPE_OK";
+export const EDIT_IMAGE= "EDIT_IMAGE";
+export const EDIT_IMAGE_FAIL ="EDIT_IMAGE_FAIL";
+export const EDIT_IMAGE_OK ="EDIT_IMAGE_OK";
+export const EDIT_CATEGORY_OK = "EDIT_CATEGORY_OK";
```

Comment:
Some action types (and associated reducer cases) appear to be missing. E.g. `EDIT_CATEGORY`, `EDIT_TAG`, `EDIT_INGREDIENT`, etc.

Reply:
i changed most of them to EDIT_ACTION_OK to differentiate between success and failure types

Reply:
Yes. But you need to initiate the action first, no?